src/widget/form/chatform.cpp

```diff
@@ -185,8 +185,12 @@ void ChatForm::onTextEditChanged()
 
 void ChatForm::onAttachClicked()
 {
-    QStringList paths = QFileDialog::getOpenFileNames(this,
-                                                      tr("Send a file"));
+    QStringList paths = QFileDialog::getOpenFileNames ( this,
+                                                      tr("Send a file"),
+                                                      QDir::homePath(),
+                                                      0,
+                                                      0,
+                                                      QFileDialog::DontUseNativeDialog );
```

Comment:
If someone can provide a Windows build of this branch then I can test it on Windows 8.1.

My experience with getting build tools working on Windows has been less than ideal.